changelog.md

```diff
@@ -0,0 +1,22 @@
+**Beta features**
+1. Multiple sidechain nodes network.
+2. Mainchain synchronization: Cross-chain Transfer Protocol support for sidechain declaration: forward and backward transfers.
```

Comment:
2. Mainchain synchronization: Cross-chain Transfer Protocol support for sidechain declaration, forward transfers, backward transfer requests and withdrawal certificates.
3. Withdrawal certificate with threshold signature zero-knowledge proof (link to zendoo-sc-cryptolib)

Reply:
done